src/js/components/Tree/stories/Tree.stories.js

```diff
@@ -0,0 +1,7 @@
+export { Busy } from './Busy';
+export { Flat } from './Flat';
+export { Nested } from './Nested';
+
+export default {
+  title: 'Controls/Tree',
```

Comment:
Are we sure it should be "Controls"? It seems like it goes where Table and List are, more so than "Controls".

Reply:
Fixed.